agent/src/beerocks/monitor/monitor_thread.cpp

```diff
@@ -1153,6 +1154,52 @@ bool monitor_thread::handle_cmdu(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx)
         mon_wlan_hal->sta_link_measurements_11k_request(mac_str);
         break;
     }
+    case beerocks_message::ACTION_MONITOR_CHANNEL_SCAN_TRIGGER_SCAN_REQUEST: {
+        auto request =
+            beerocks_header
+                ->addClass<beerocks_message::cACTION_MONITOR_CHANNEL_SCAN_TRIGGER_SCAN_REQUEST>();
+        if (!request) {
+            LOG(ERROR) << "addClass cACTION_MONITOR_CHANNEL_SCAN_TRIGGER_SCAN_REQUEST failed";
+            return false;
+        }
+
+        auto radio_mac         = request->scan_params().radio_mac;
+        auto dwell_time_ms     = request->scan_params().dwell_time_ms;
+        auto channel_pool      = request->scan_params().channel_pool;
+        auto channel_pool_size = request->scan_params().channel_pool_size;
+        auto channel_pool_vector =
+            std::vector<unsigned int>(channel_pool, channel_pool + channel_pool_size);
+        std::string channels;
+        for (int8_t index = 0; index < channel_pool_size; index++) {
```

Comment:
Use regular `int` instead of `int8_t`
Also, add a newline separation between variable declaration and code segments

Reply:
This loop is for debug purposes only? If so, please add a relevant comment

Reply:
Changed the `uint8_t` to `int`.
Added a comment explaining this loop is only for the print.
And made the whole loop thinner by using a ternary operator instead of the if statement
`channels += ((index != 0) ? "," : "") + std::to_string(int(channel_pool[index]));`